python/dgl/graphbolt/impl/sampled_subgraph_impl.py

```diff
@@ -179,6 +221,8 @@ def exclude_edges(
         "The sampled subgraph and the edges to exclude should be both "
         "homogeneous or both heterogeneous."
     )
+    # Get edges to exclude.
+    edges = _find_exclude_edges(edges, exclude_mode, reverse_etypes)
```

Comment:
The name `_find_exclude_edges` is confusing. It is more like expanding or transforming the edge set to exclude. Moreover, I think the logic of this function is independent from `exclude_edges`. What about we call it before `exclude_edges`?